src/features/plan/Plan.js

```diff
@@ -1,70 +1,254 @@
+                <Card
+                    isCentered
+                    heading="No transactions to display."
+                    subheading="There is no spending transaction in your account(s) to import."
+                />
+            );
+        } else {
+            return (
+                <Card heading="Imported spending transactions">
+                    <Card isNested>
+                        <Table values={transactions} />
+                    </Card>
+                </Card>
+            );
+        }
+    };
+
+    const DeleteBudgetCard = () => {
+        return (
+            <Card isCentered heading="Delete your budget.">
+                <NavButton to="./delete-budget" text="Delete" bg="dim" c="fg" />
+            </Card>
+        );
+    };
+
+    if (budget === 0) {
+        return (
+            <PageTemplate page="plan">
+                <Breadcrumbs
+                    path={"Home/Budget Planner"}
+                    links={["/dashboard", "/plan"]}
+                />
+                <Grid
+                    w="100%"
+                    gap="25px"
+                    autoColumns="minmax(600px, auto)"
+                    autoFlow="row"
+                >
+                    <GridItem>
+                        <IntroductionCard />
+                    </GridItem>
+                </Grid>
+                <Outlet />
+            </PageTemplate>
+        );
+    } else {
+        return (
+            <PageTemplate page="plan">
+                <Breadcrumbs
+                    path={"Home/Budget Planner"}
+                    links={["/dashboard", "/plan"]}
+                />
+                <Grid
+                    w="100%"
+                    gap="25px"
+                    autoColumns="minmax(600px, auto)"
+                    autoFlow="row"
+                >
+                    <GridItem>
+                        <Card heading="Budget Details">
+                            <Card isNested>
+                                <LightTable
+                                    headers={[
+                                        "Start Date",
+                                        "End Date",
+                                        "Duration",
+                                    ]}
+                                    primary={[
+                                        format(
+                                            parseISO(start_date),
+                                            "dd LLLL yyyy"
+                                        ),
+                                        format(
+                                            parseISO(end_date),
+                                            "dd LLLL yyyy"
+                                        ),
+                                        `${differenceInDays(
+                                            parseISO(end_date),
+                                            parseISO(start_date)
+                                        )} days`,
```

Comment:
`date-fns` has the function `formatDistanceStrict` which takes care of the unit (days, months, years) as well as plurality - https://date-fns.org/v2.28.0/docs/formatDistanceStrict

That way you won't end up with phrases like `1 days`.